src/rules/hindi.toml

```diff
@@ -0,0 +1,19 @@
+min_trimmed_length = 0
+min_word_count = 0
+max_word_count = 50
+min_characters = 0
+may_end_with_colon = false
+quote_start_with_letter = false
+needs_punctuation_end = false
+needs_letter_start = false
+needs_uppercase_start = false
+matching_symbols = [
+  ["।", "."]
+]
+replacements = [
+  ["।", "."]
+]
+disallowed_symbols = [
+  'a', 'e', 'i', 'o',
```

Comment:
I assume this tries to make sure that there are no latin script words that would be foreign to Hindi? Would it be easier to define a regex for all `allowed_characters` then?

If not, you're missing `u` ;)

Reply:
Going the `allowed_characters` way makes the regex a bit complicated. Oops I missed the `u`. Thanks. Updated.